vowpalwabbit/io_item.h

```diff
@@ -0,0 +1,51 @@
+#ifndef _IO_ITEM_
+#define _IO_ITEM_
+class IO_Item {
+
+  public:
+
+      std::string message;
+      int numCharsInit;
```

Comment:
I think numCharsInit is unnecessary now. IIRC during our call we figured out that you could detect this while reading the file and just set done to true instead of saving it.